trackersite/tracker/templates/tracker/ticket_table.html

```diff
@@ -1,33 +1,99 @@
 {% load i18n trackertags %}
-<table class="ticket_list">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
+<style>
+    .container {
+        width: 100% !important;
+    }
+</style>
+<script type="text/javascript">
+
+$( document ).ready(function() {
+	$( "i" ).each(function() {
+		//console.log($( this ).parent().attr("onclick").split("'")[1]);
+		if( $( this ).parent().attr("onclick").split("'")[1] == getUrlParameter("order") && getUrlParameter("state") == "1") {
+			$( this ).addClass("fa-caret-up");
+			console.log("state 1")
+		} else if( $( this).parent().attr("onclick").split("'")[1] == getUrlParameter("order") && getUrlParameter("state") == "2") {
+			$( this ).addClass("fa-caret-down");
+		} else {
+			$( this ).addClass("fa-sort");
+		}
+	});
+})
+var getUrlParameter = function getUrlParameter(sParam) {
+    var sPageURL = decodeURIComponent(window.location.search.substring(1)),
+        sURLVariables = sPageURL.split('&'),
+        sParameterName,
+        i;
+
+    for (i = 0; i < sURLVariables.length; i++) {
+        sParameterName = sURLVariables[i].split('=');
+
+        if (sParameterName[0] === sParam) {
+            return sParameterName[1] === undefined ? true : sParameterName[1];
+        }
+    }
+};
+
+var state = 0;
+function order(by) {
+    state = getUrlParameter('state');
+    if (state == '0') {state = 0;}
+    if (state == '1') {state = 1;}
```

Comment:
Tahle celá věc jednak patří do externího souboru, za druhé: fakt potřebujeme něco co popisuje elementy `<i>` a obsahuje řádky jako `if (state == '0') {state = 0;}`? Nešlo by to nějak zpříčetnit?

Reply:
Elementy `<i>` jsou ikony do font awesome, jelikož tyhle tagy se nemají používat, font awesome si je trochu přejmenoval z italics na icon ;). 

Nějak to zpříčetnit asi půjde, pokusím se vymyslet jiné řešení.